packages/nimble-components/src/table/components/row/template.ts

```diff
@@ -40,17 +43,29 @@ export const template = html<TableRow>`
         `)}
         <span class="row-front-spacer ${x => (x.isTopLevelParentRow ? 'top-level-parent' : '')}"></span>
         ${when(x => x.isParentRow, html<TableRow>`
-            <${buttonTag}
-                appearance="${ButtonAppearance.ghost}"
-                content-hidden
-                class="expand-collapse-button"
-                tabindex="-1"
-                @click="${(x, c) => x.onRowExpandToggle(c.event)}"
-                title="${x => (x.expanded ? tableRowCollapseLabel.getValueFor(x) : tableRowExpandLabel.getValueFor(x))}"
-                aria-hidden="true"
-            >
-                <${iconArrowExpanderRightTag} ${ref('expandIcon')} slot="start" class="expander-icon ${x => x.animationClass}"></${iconArrowExpanderRightTag}>
-            </${buttonTag}>
+            ${when(x => x.loading, html<TableRow>`
+                <span class="spinner-container">
```

Comment:
I presume this container element is providing the same layout behaviors for the spinner element that the expander `button` element provides for its content? Basically I'm just fishing for why this container element is necessary.

Reply:
Yes, your assumption is correct. I create the container so that I can size & position it exactly as the expander button. Then, I put the spinner centered in the container so that I know that the spinner will be positioned where the expand/collapse icon is within the button.